### PR TITLE
lanetix/issues#2686: bump eslint versions. add babelify support. clea…

### PR DESCRIPTION
**Pull request by scarney81**
@lanetix/engineers - review please

- bumped eslint versions
- added support for babelify (which reactifies the code - hence the removal)
- needed to clear the interval function to avoid generating exceptions in the console
- fix lint errors

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -42,10 +42,7 @@
     "no-shadow": 0,
     "dot-notation": 2,
     "no-underscore-dangle": 2,
-    "space-after-keywords": [2, "always", {
-      "checkFunctionKeyword": true
```
Comment (apechimp) [inline .eslintrc]: Why remove the function check?

Comment (scarney81) [inline .eslintrc]: It breaks in the new eslint

Comment (scarney81) [inline .eslintrc]: They've moved it to it's own rule: http://eslint.org/blog/2015/02/eslint-0.15.0-released/#new-rule-space-before-function-parentheses

I'll update this to include the new rule-io

File: package.json [hunk 2]
```diff
@@ -16,16 +16,16 @@
   "browserify": {
     "transform": [
       "cssify",
-      ["reactify", { "es6": true }]
+      "babelify"
```
Comment (apechimp) [inline package.json]: You need to adjust your dependencies for this to work. Also, why switch to babel? I don't see any ES6...

File: .eslintrc (hunk 1 above)
Comment (scarney81) [inline .eslintrc]: @apechimp - https://github.com/lanetix/react-horizontal-scroll/commit/d5c28b7bf87b59d1e228a579c7f0ca5674aeabab#diff-1dc6ee56b778cd91e0327b52aaeaa1b9R46

Comment (apechimp) [inline .eslintrc]: Ahh, nice! FWIW, I've started using [the js standard style](https://github.com/feross/standard) for all my non-work things and noticed it's nice to stop worrying about all this :hankey:.

File: package.json (hunk 2 above)
Comment (scarney81) [inline package.json]: I don't believe I do, I've been able to verify this locally.

File: .eslintrc (hunk 1 above)
Comment (scarney81) [inline .eslintrc]: Nice!

**Comment by apechimp** [comment]
LGTM :carousel_horse:

File: package.json (hunk 2 above)
Comment (scarney81) [inline package.json]: I re-jiggered these fwiw

**Pull request merged**